hexrdgui/hexrd_config.py

```diff
@@ -402,6 +403,7 @@ def _attributes_to_persist(self):
             ('custom_polar_tth_distortion_object_serialized', None),
             ('_previous_structureless_calibration_picks_data', None),
             ('sample_tilt', [0, 0, 0]),
+            ('recent_state_files', []),
```

Comment:
We will want this `recent_state_files` to be saved/loaded in QSettings, but not in state files. We might have to add some code to skip it when loading state files.

Reply:
We probably also want to make sure they aren't saved to state files either, even if we don't load them. We probably don't want paths in the user's filesystem unknowingly written to a file that they will send around.